.github/workflows/build.yml

```diff
@@ -90,3 +90,16 @@ jobs:
           platforms: linux/amd64
           push: true
           tags: ${{ steps.image-name.outputs.name }}
+
+      - name: Scan Docker image
+        id: container-scan
+        uses: anchore/scan-action@3343887d815d7b07465f6fdcd395bd66508d486a # v3.6.4
```

Comment:
@withinfocus should we bump this [action](https://github.com/anchore/scan-action/releases/tag/v4.1.0) up to v4.1.0

Reply:
I try to catch these usually and didn't notice this one, but Renovate will handle it via https://github.com/bitwarden/key-connector/pull/165.